Mage.Tests/src/test/java/org/mage/test/utils/ManaOptionsTest.java

```diff
@@ -396,9 +377,8 @@ public void testCabalCoffers1() {
         execute();
 
         ManaOptions manaOptions = playerA.getAvailableManaTest(currentGame);
-        assertDuplicatedManaOptions(manaOptions);
 
-        Assert.assertEquals("mana variations don't fit", 1, manaOptions.size());
+        Assert.assertEquals("mana variations don't fit", 1, manaOptions.size()); // TODO: Why one, should there be {B} and {B}{U}?
```

Comment:
Why did this pass below. Shouldn't the available options be {B}{U} (tapping basic lands) and {B} (paying {B}{U} to tap Cabal Coffers and get a single {B} back)?